src/gui/gtk.c

```diff
@@ -1550,36 +1550,36 @@ void dt_gui_gtk_run(dt_gui_gtk_t *gui)
   dt_cleanup();
 }
 
+// refactored function to read current ppd, because gtk for osx has been unreliable
+// we use the specific function here. Anyway, if nothing meaningful is found we default back to 1.0
+double dt_get_system_gui_ppd(GtkWidget *widget)
+{
+  double res = 0.0f;
+#ifndef GDK_WINDOWING_QUARTZ
```

Comment:
We have a huge number of #ifdef for this and 4 #ifndef. I'm proposing to use #ifdef here, easier to read to me.

Reply:
right